test/EscrowERC20.t.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {Test, console} from "forge-std/Test.sol";
+import {EscrowERC20} from "../src/EscrowERC20.sol";
+import "./mocks/MockERC20.sol";
+
+contract EscrowERC20Test is Test {
+    EscrowERC20 public escrowERC20;
+    MockERC20 public mockERC20;
+    uint256 totalSupply = 10 ** 18;
```

Comment:
Nitpick - Total supply is just 1 token? Because default decimal places in an ERC20 contract is 18
```suggestion
    uint256 totalSupply = 1000000 * (10 ** 18);
```